go.mod

```diff
@@ -35,6 +21,10 @@ require (
 	github.com/bazelbuild/buildtools v0.0.0-20200922170545-10384511ce98
 	github.com/blang/semver/v4 v4.0.0
 	github.com/bwmarrin/snowflake v0.0.0
+	// Upstream is unmaintained. This fork introduces two important changes:
+	// * We log an error if writing a cache key fails (e.G. because disk is full)
```

Comment:
Nit: e.g.

Reply:
Oops forgot to fix this. I've addressed it in the next PR.